homeassistant/components/androidtv/media_player.py

```diff
@@ -304,12 +310,17 @@ def adb_command(self, cmd):
         """Send an ADB command to an Android TV / Fire TV device."""
         key = self._keys.get(cmd)
         if key:
-            return self.aftv.adb_shell('input keyevent {}'.format(key))
+            self.aftv.adb_shell('input keyevent {}'.format(key))
+            self._adb_response = ''
```

Comment:
Is there no response for this (for example, if the call succeeded or not)?

Reply:
Right, there's no response.